spec/api/apitest.js

```diff
@@ -0,0 +1,29 @@
+'use strict';
+
+const request = require('supertest');
+const app = require('../../server/server');
+
+describe('GET /api/proposals', () => {
+  it('respond with json', (done) => {
+    request(app)
+      .get('/api/proposals')
+      .set('Accept', 'application/json')
+      .expect('Content-Type', /json/, done)
+      .expect(200, done);
```

Comment:
You may want to test that the response is an array of objects with the correct structure. That may involve creating some test data and then cleaning it up after the test. Ideally you would programmatically associate the proposal model with the memory datasource so don't have to worry about cleaning up the test data.